app/views/shared/_post.html.erb

```diff
@@ -0,0 +1,5 @@
+<div class="post">
+  <h3>Post #<%= post.id %></h3>
+  <p><%= post.text %></p>
+  <p>Comments: <%= post.comments_counter %>, Likes: <%= post.likes_counter %> </p>
+</div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add a feature in your application when a User clicks on each `Post` takes him to their `show page` as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png). I suggest you apply the same approach you did in `app/views/shared/_user.html.erb` file.